test/tests.zig

```diff
@@ -140,6 +140,14 @@ const test_targets = blk: {
             .use_llvm = false,
             .use_lld = false,
         },
+        .{
+            .target = std.Target.Query.parse(.{
+                .arch_os_abi = "spirv64-vulkan",
+                .cpu_features = "vulkan_v1_2+int8+int16+int64+float16+float64",
```

Comment:
Why the additional features? (Should they be rolled into `vulkan_v1_2`?)

Reply:
no they are not always available but we temporarily enable them because composite ints hasn't implemented yet

Reply:
> Also, do we want spirv64-opencl in the matrix? spirv64-opengl?

yes i'd say very soon. i picked vulkan first because getting a working opencl runtime on windows is painful. not sure about opengl. perhaps in future